tasks.rb

```diff
@@ -2307,6 +2321,34 @@ def set_hpxml_hvac_distributions(hpxml_file, hpxml)
                                           duct_insulation_r_value: 0,
                                           duct_location: HPXML::LocationAtticUnvented,
                                           duct_surface_area: 50)
+  elsif ['base-misc-defaults.xml'].include? hpxml_file
+    hpxml.hvac_distributions[0].ducts[0].duct_surface_area = nil
+    hpxml.hvac_distributions[0].ducts[1].duct_surface_area = nil
+    hpxml.hvac_distributions[0].ducts[0].duct_location = nil
+    hpxml.hvac_distributions[0].ducts[1].duct_location = nil
+  elsif ['base-misc-defaults3.xml'].include? hpxml_file
+    hpxml.hvac_distributions[0].ducts.each { |duct| duct.duct_surface_area = nil }
+    hpxml.hvac_distributions[1].ducts.each { |duct| duct.duct_surface_area = nil }
+    hpxml.hvac_distributions[4].ducts.each { |duct| duct.duct_surface_area = nil }
+    hpxml.hvac_distributions[5].ducts.each { |duct| duct.duct_surface_area = nil }
+    hpxml.hvac_distributions[0].ducts.each { |duct| duct.duct_location = nil }
+    hpxml.hvac_distributions[1].ducts.each { |duct| duct.duct_location = nil }
+    hpxml.hvac_distributions[4].ducts.each { |duct| duct.duct_location = nil }
+    hpxml.hvac_distributions[5].ducts.each { |duct| duct.duct_location = nil }
+  elsif ['invalid_files/missing-duct-location-and-surface-area.xml'].include? hpxml_file
+    hpxml.hvac_distributions[0].ducts[1].duct_surface_area = nil
+    hpxml.hvac_distributions[0].ducts[1].duct_location = nil
+    hpxml.hvac_distributions[1].ducts[1].duct_surface_area = nil
+    hpxml.hvac_distributions[1].ducts[1].duct_location = nil
+    hpxml.hvac_distributions[4].ducts[1].duct_surface_area = nil
+    hpxml.hvac_distributions[4].ducts[1].duct_location = nil
+    hpxml.hvac_distributions[5].ducts[1].duct_surface_area = nil
+    hpxml.hvac_distributions[5].ducts[1].duct_location = nil
+  elsif ['invalid_files/missing-duct-location.xml'].include? hpxml_file
+    hpxml.hvac_distributions[0].ducts[1].duct_location = nil
+    hpxml.hvac_distributions[1].ducts[1].duct_location = nil
+    hpxml.hvac_distributions[4].ducts[1].duct_location = nil
+    hpxml.hvac_distributions[5].ducts[1].duct_location = nil
```

Comment:
Surely this can be done more programmatically? Either by looping through all have distributions and checking the type, or looping through hvac distribution indices of interest.